pkg/hookbot/hookbot.go

```diff
@@ -310,16 +324,20 @@ func (h *Hookbot) Del(l Listener) {
 
 // The topic is everything after the "/pub/" or "/sub/"
 // Do not capture the "/unsafe". See note in `Topic()`.
-var TopicRE *regexp.Regexp = regexp.MustCompile("^(?:/unsafe)?/[^/]+/(.*)$")
+var TopicRE *regexp.Regexp = regexp.MustCompile("^(?:/unsafe)?/(?:pub|sub)/(.*)$")
```

Comment:
I am confused :eyes:. What is the third case? Right now I see:

* /pub/foo
* /sub/foo
* /unsafe/foo
* /unsafe/foo
* /foo

I don't see how to distinguish the last case from a typo?